requirements.txt

```diff
@@ -1,12 +1,16 @@
 Django==4.2.3
+wheel==0.41.1
+setuptools==68.1.0
+psycopg2>=2.8
+psycopg2-binary==2.9.7
+GDAL==3.7.1.1
```

Comment:
The GDAL package is causing issues in github actions. That could be resolved, but locustempus isn't actually using the gdal python package, so this shouldn't be required. Any gdal errors that we run into in docker should be resolved by just installing the `libgdal-dev` package at the system level.